governance/dapps_dev.json

```diff
@@ -228,7 +228,18 @@
     "dapps": [
       {
         "title": "Polkassembly",
-        "url": "https://moonriver.polkassembly.io/referendum/{referendumId}",
+        "url": "https://moonriver.polkassembly.io/referenda/{referendumId}",
```

Comment:
think we need to move it to:  governance/v2/dapps_dev.json

Reply:
looks like we already have that in dapps_dev - removed it